scripts/user-actions/remove.js

```diff
@@ -3,24 +3,24 @@ console.log('--- loading: remove.js');
 
 const removeHandler = () => {
   debugger;
-  const userInput = prompt('');
-  if () {
+  const userInput = prompt('Enter a word to remove: ');
+  if (userInput === null) {
     return;
   }
 
   const isUserInput = (entry) => {
-    return _;
+    return entry === userInput;
   };
   const inputIsInArray = words
-    ._(_);
+    .find(isUserInput);
```

Comment:
Hi @okozmovskaya Olga, I tested your codes and it worked here but just one small thing: it has no problem but is good that line 14 and 15 are in one line, like this: 
```js 
const inputIsInArray = words.find(inUserInput); 
``` 
But if you dont change is it , it is still ok and it works!

Greetings, Sayed

Reply:
Thank you, Sayed for your checking and advise!